### ISSUE TITLE
Dimensional Travel affects fist use as a tool to break blocks

### ISSUE THREAD

**Issue by Dragnier** [rwtema/modjam2018#7]
Traveling to any dimension tends to make blocks break super slow. Exit to menu and restart usually fixes it.

**Comment by darthvader45** [comment]
Agreed, this and a few other bugs need to be fixed, as they cause issues that completely break the game.  I've lost so much progress doing this.  I really need to progress, and having to revert to a backup every time I glitch out, is a real problem.

**Comment by dalgrim** [comment]
I can confirm this on the Direwolf20 2.2 pack with monk mod added.
going to the nether and break a block bare handed causes the block to desync. it removes it on screen but remains as invisible in game. Flight does not work there, and armor icons do not show up.
